generator/src/test/java/com/scottlogic/deg/generator/cucumber/utils/CucumberTestState.java

```diff
@@ -38,14 +38,14 @@
 
```

Comment:
There are no functional changes to this class. Be careful of committing format-only changes in non-related classes as it can cause merge conflicts with someone that actually wanted to affect this class.